library/general/containerlimits/template.yaml

```diff
@@ -172,6 +172,10 @@ spec:
           general_violation[{"msg": msg, "field": "initContainers"}]
         }
 
+        violation[{"msg": msg}] {
+          general_violation[{"msg": msg, "field": "ephemeralContainers"}]
```

Comment:
may have one question
It is said `Pod resource allocations are immutable, so setting resources is disallowed.` in
https://kubernetes.io/docs/concepts/workloads/pods/ephemeral-containers/#what-is-an-ephemeral-container

So, do we need to add ephemeralContainer check in container limit policy?